### PR TITLE
Update readme and fixes #65

### LINKED ISSUE #65: Update readme with details on how to run software on your local.
I found this one missing from our README.md. and I think it is very important to have if anyone new would like to contribute. so it would be great if we can update readme for that.

### PR DESCRIPTION
**Pull request by Shivank98**
i tried to add screenshots folder in .gitignore but, it didn't.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,6 @@ target/
 
 .idea/
 venv/
+
+# Folder
+./screenshots
```
Comment (theSage21) [inline .gitignore]:
```suggestion
screenshots/
```

File: README.md [hunk 2]
```diff
@@ -77,3 +77,40 @@ To review:
 2. Go through the code.
 3. If something is not clear, ask for clarification. If someone has already asked just thumbs up that comment.
 4. Add an approving comment saying "Good work!". If someone has already commented thumbs up on that.
+
+## How To Contribute and Run Software
+* Fork This Repo by clicking on button as shown in image.
+![Fork](screenshots/1.png)
```
Comment (theSage21) [inline README.md]: if you have added screenshots to the gitignore then no images will be there as reference to.

File: README.md [hunk 3]
```diff
@@ -77,3 +77,40 @@ To review:
...
+
+* now you sould have a copy of Repo for yourself with `username/pyjudge`.
+![Clone](screenshots/2.png)
+
```
Comment (theSage21) [inline README.md]: any reason extra blank lines are there?

File: README.md [hunk 4]
```diff
@@ -77,3 +77,40 @@ To review:
...
+* after this you will need to clone from the Green tab you can see in above image.
+
+* now let's come to your local machine.
+    * open terminal
+    * create a folder "PyJudge" by `$ mkdir PyJudge`.
```
Comment (theSage21) [inline README.md]: the next 3 steps are not needed. Clone will create the folder.

File: README.md [hunk 5]
```diff
@@ -77,3 +77,40 @@ To review:
...
+    * go to folder `$ cd PyJudge`.
+    * initialize the git by ` $ git init`.
+    * `$ git clone https://github.com/YourUsername/PyJudge.git`
+    * now you have a folder called PyJudge which has all your code files.
+    * now go to that folder `cd PyJudge`.
+    * create a virtual enviroment `virtualenv venv`.
```
Comment (theSage21) [inline README.md]: I would recommend using pipenv instead of this.

File: README.md [hunk 6]
```diff
@@ -77,3 +77,40 @@ To review:
...
+    * activate your enviroment `source venv/bin/activate`.
+    * install dependencies `pip install -r requirements.txt`.
+    * and run the Software by `python server.py`.
+
+    ### To Contribute
+    * follow the above setup once.
+    *  create a new github branch in `/PyJudge/PyJudge` by `git checkout -b BranchName`
+    * now make changes in your files.
+    * when done see changes from `git status`. it will show all file which you updated or modified.
+    * add all those file to your git repo by `git add .`. `.` dot will add all files if you want a perticuler file to be added use, `git add filename`
```
Comment (theSage21) [inline README.md]: I recommend using `git add -Av` easier to catch errors than `git add .`

**Comment by theSage21** [comment]
I've added required pipfiles in #67

**Comment by Shivank98** [comment]
How do you know so much :D
will make changes by tomorrow?

File: README.md (hunk 2 above)
Comment (Shivank98) [reply README.md]: so should I remove screenshots? or we should keep it ?

File: README.md (hunk 6 above)
Comment (Shivank98) [reply README.md]: do you mean `git add -A` ?

**Comment by theSage21** [comment]
Av

That's all and verbose

**Comment by Shivank98** [comment]
@theSage21 please review it when you get time.

**Pull request merged**